upstream-community-operators/starboard-operator/0.0.1/starboard-operator.v0.0.1.clusterserviceversion.yaml

```
@@ -0,0 +1,316 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  name: starboard-operator.v0.0.1
+  namespace: starboard-operator
+  annotations:
+    capabilities: Basic Install
+    categories: Security
+    description: Keeps Starboard resources updated
```

```suggestion
    description: Keeps Starboard security report resources updated
```

I'm wondering if "Keeps security report resources updated" might be even better - Starboard is already in the name, and it might slightly hint towards these being generic security reports. Wdyt?

---

Yes I like it without the Starboard word. Already updated.